cartographer_ros/cartographer_ros/trajectory_options.cc

```diff
@@ -50,6 +50,12 @@ TrajectoryOptions CreateTrajectoryOptions(
   options.provide_odom_frame =
       lua_parameter_dictionary->GetBool("provide_odom_frame");
   options.use_odometry = lua_parameter_dictionary->GetBool("use_odometry");
+  options.fixed_frame_pose_frame =
```

Comment:
I don't think this should be an option. Analogous to IMU, there is an input topic "fixed_frame_pose". We can re-map in the launch file if a different name is needed.

Reply:
Done.